wdl/pipelines/PacBio/VariantCalling/PBCCSWholeGenome.wdl

```diff
@@ -4,6 +4,7 @@ import "../../../tasks/Utility/PBUtils.wdl" as PB
 import "../../../tasks/Utility/Utils.wdl" as Utils
 import "../../../tasks/VariantCalling/CallVariantsPBCCS.wdl" as VAR
 import "../../../tasks/Utility/Finalize.wdl" as FF
+import "../../../tasks/VariantCalling/TRGT.wdl" as TRGT
```

Comment:
I have a general comment about including `trgt` in this production pipeline.
It's a bit complicated, so I'll outline the main logic here:

The production pipeline is intended for running routine and stable tools on all production samples. Its primary responsibility is two-fold: alignments merging (and associated QC/metric) + variant calling (SV, SNV, indel). The tools incorporated here are well accepted by the community and undergo less frequent updates. This means we don't need to update & rerun the workflow constantly.

There's another set of callers that are not quite "validated" yet, i.e. they are more special-purpose focused and/or target a specific type of variants. They are under active development and evaluation so their release frequencies are much higher. IMO `trgt` falls under this category.

So here's my proposal:
Let's leave `trgt` out of the main variant calling pipeline, and promote it to a standalone `workflow` (i.e. place it under `/wdl/pipelines/PacBio/VariantCalling/`). We have other means to ensure that such "special-purpose" callers are also run routinely, in addition to the main ones. This also has the benefit that if we decide to update the version of `trgt` or how to run it (added annotations, etc), we only need to update its WDLs and rerun that workflow, i.e. [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) .

How does this sound?

Reply:
You make an excellent point. I definitely agree that TRGT is much more 'under active development' than may be ideal for the main pipeline.

I agree with your plan and have gone ahead and created a new pipeline under /wdl/pipelines/PacBio/VariantCalling/ which is called PBCCS_CallTRs.wdl. This pipeline runs the processWithTRGT task in the TRGT.wdl file (under /wdl/tasks/VariantCalling/).